docs/charts/profile_columns.ipynb

```diff
@@ -0,0 +1,551 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# `profile_columns`\n",
+    "\n",
+    "!!! info \"At a glance\"\n",
+    "    **Useful for:** Looking at the distribution of values in columns. \n",
+    "\n",
+    "    **API Documentation:** [profile_columns()](../../linker.md#splink.linker.Linker.profile_columns)\n",
```

Comment:
The link appears to be broken

Reply:
Ah yea, I have moved things around and I think a few of the API docs links are broken - I will do a sweep of them now

Reply:
fix pushed

Reply:
Thank you! I'll stop commenting on them and assume they're all fixed.